src/components/engineService.ts

```diff
@@ -275,16 +291,8 @@ export class EngineService {
 	}
 
 	public async __getCodemodNames(): Promise<ReadonlyArray<string>> {
-		const executableUri = this.__codemodEngineNodeExecutableUri;
-
-		if (executableUri === null) {
-			throw new EngineNotFoundError(
-				'The codemod engine node has not been downloaded yet',
-			);
-		}
-
 		const childProcess = spawn(
-			singleQuotify(executableUri.fsPath),
+			this.__getCodemodEngineNodeExecutableCommand(),
```

Comment:
will it always deliver a correct path?

Reply:
it can throw if engineNode is not bootstrapped, in other case it should return exactly same path as previously